docs/development/running-e2e-tests.md

```diff
@@ -2,194 +2,96 @@
 title: Local e2e Tests
 ---
 
-# e2e Test Suite
+# Go-native e2e Test Suite
 
-Developers can run extended e2e tests, in addition to unit tests, for Etcd-Druid in or from 
-their local environments. This is recommended to verify the desired behavior of several features
-and to avoid regressions in future releases.
+The Etcd-Druid project now features a comprehensive, Go-native e2e test suite (see [PR #1060](https://github.com/gardener/etcd-druid/pull/1060)) that comprehensively covers various aspects of Etcd-Druid's management of Etcd clusters. The suite is designed for reliability, speed, and ease of use, and does not require skaffold or any provider-specific setup.
 
-The very same tests typically run as part of the component's release job as well as on demand, e.g.,
-when triggered by Etcd-Druid maintainers for open pull requests.
+## Running e2e tests against an existing cluster and etcd-druid
```

Comment:
Another section detailing how to spin up a cluster for the e2e tests is necessary as `DRUID_E2E_TEST=true` seems to be necessary with `make kind-up` for the parameters in `skaffold` to be set appropriately for the tests.